### PR TITLE
feat(krumbs): redesign krumbs

### PR DESCRIPTION
**Pull request by aanchalm01**
### Summary
Update `Krumbs` to new metalab designs.
#### Changes made:
*

### PR Checklist
- [ ] Does not introduce dependencies
- [ ] **Functional:** all changes do not break existing APIs and if so, bump major version.
- [ ] **Tests pass:** check the output of yarn test packages/<Kongponent>
- [ ] **Naming:** the files and the method and prop variables use the same naming conventions as other Kongponents
- [ ] **Framework style:** abides by the essential rules in Vue's style guide
- [ ] **Cleanliness:** does not have formatting issues, unused code (e.g., console.logs), or leftover comments
- [ ] **Docs:** includes a technically accurate README, uses JSDOC where appropriate

### CODE REVIEW

File: packages/Krumbs/Krumbs.vue [hunk 1]
```diff
@@ -63,20 +67,22 @@ export default {
...
 
-.krumbs .krumb-item:last-of-type:after,
-.krumbs .krumb span:after {
-  display: inline-block;
-  padding-right: 0.5rem;
-  padding-left: 0.5rem;
-  color: #636c72;
-  content: "\203A";
+.krumbs li a {
+  color: #6F7787;
```
Comment (kaiarrowood) [inline packages/Krumbs/Krumbs.vue:79]:
```suggestion
  color: var(--grey-500);
```

File: packages/Krumbs/Krumbs.vue [hunk 2]
```diff
@@ -63,20 +67,22 @@ export default {
...
-.krumbs .krumb-item + .krumb-item::before {
+.krumbs .krumb-item  svg {
   display: inline-block;
-  padding-right: 0.5rem;
-  padding-left: 0.5rem;
-  color: #636c72;
-  content: "\203A";
+  padding-right: 14px;
+  padding-left: 14px;
+  color: #6F7787;
```
Comment (kaiarrowood) [inline packages/Krumbs/Krumbs.vue:74]:
```suggestion
  color: var(--grey-500);
```

File: packages/Krumbs/Krumbs.vue [hunk 3]
```diff
@@ -20,6 +20,10 @@
         :title="item.title"
         :href="item.to"
         target="_blank">{{ item.text }}</a>
+      <KIcon
+        icon="chevronRight"
+        size="15"
+        color="#6F7787"/>
```
Comment (kaiarrowood) [inline packages/Krumbs/Krumbs.vue:26]:
```suggestion
        color="var(--grey-500)"/>
```

File: packages/Krumbs/Krumbs.vue [hunk 4]
```diff
@@ -63,20 +67,22 @@ export default {
...
 }
 
-.krumbs .krumb-item + .krumb-item::before {
+.krumbs .krumb-item  svg {
   display: inline-block;
-  padding-right: 0.5rem;
-  padding-left: 0.5rem;
-  color: #636c72;
-  content: "\203A";
+  padding-right: 14px;
```
Comment (kaiarrowood) [inline packages/Krumbs/Krumbs.vue:72]:
```suggestion
```
Make it one line

File: packages/Krumbs/Krumbs.vue [hunk 5]
```diff
@@ -63,20 +67,22 @@ export default {
...
+  padding-left: 14px;
```
[the remark this answers is not in the document]
Comment (aanchalm01) [reply packages/Krumbs/Krumbs.vue]: I was checking the value in an incorrect manner. Thanks for adding the screenshot! @kaiarrowood 
Screen Shot 2021-09-30 at 11 32 21 AM

**Review by kaiarrowood — APPROVED** [review]
Looks great! 🐉 🔥

**Pull request merged**